common/constants.go

```diff
@@ -129,11 +129,13 @@ const (
 	ArchivalPaused = "paused"
 )
 
-// Queue types used in queue table
-// Use positive numbers for queue type
-// Negative numbers are reserved for DLQ
 const (
-	DomainReplicationQueueType QueueType = 1
+	// TransferQueueType is the type for transfer queue
```

Comment:
common/constants seems like a weird place for these consts. Where are they being used outside of history / persistence ? If these are only used in history/persistence, consider moving these to either history or persistence datainterfaces